Dockerfile

```
@@ -101,10 +101,13 @@ RUN set -eux; \
 # Dev image
 FROM app_php AS app_php_dev
 
-RUN apk add --no-cache bash
-RUN curl -1sLf 'https://dl.cloudsmith.io/public/symfony/stable/setup.alpine.sh' | bash
-RUN apk add symfony-cli
-RUN apk add --no-cache make
+RUN apk add --no-cache \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)